src/plugins/score-plugin-midi/Midi/MidiDrop.hpp

```diff
@@ -18,11 +18,101 @@ class DropHandler final : public Process::ProcessDropHandler
       const score::DocumentContext& ctx) const noexcept override;
 };
 
+struct PitchbendData{
+  int16_t bend;
+};
+struct ControllerData {
+  midi_size_t channel;
+  midi_size_t number;
+  midi_size_t value;
+  static ControllerData make_cc(const midi_size_t channel, const midi_size_t controller, const midi_size_t value)
+  {
+    SCORE_ASSERT(channel >= 0 && channel < 16);
+    return ControllerData{channel, controller, value};
+  }
+};
+
+struct NoteOnData {
+  midi_size_t channel;
+  midi_size_t note;
+  midi_size_t velocity;
+};
+struct NoteOffData {
+  midi_size_t channel;
+  midi_size_t note;
+  midi_size_t velocity;
+};
+struct MidiTrackEvent {
+  static MidiTrackEvent make_note_off(const double start, const midi_size_t ch, const midi_size_t n, const midi_size_t v){
+    return MidiTrackEvent{m_start: start, m_message:  Midi::NoteOffData{channel: ch, note: n, velocity: v}};
```

Comment:
{m_start: start,

is not valid C++ syntax AFAIK, it should be 

     .m_start = start

 :)

Reply:
<img width="439" alt="image" src="https://user-images.githubusercontent.com/87944/168427180-c4b2f5df-8f65-4853-9ee5-e54b4c7111a7.png">

Mmmh, ok, learning that I like GNU old-style (probably useful when C/C++ didn't agree yet on non positional saving lives in codebases terms) 🙂 

How can I make sure my PR isn't going to add warnings?